src/foam/nanos/controller/ApplicationController.js

```diff
@@ -137,6 +137,16 @@ foam.CLASS({
   `,
 
   properties: [
+    {
+      name: 'loginVariables',
+      expression: function(client$userDAO) {
+        return {
+          dao_: client$userDAO || null,
+          imgPath: '',
+          group: 'system'
+        };
+      }
```

Comment:
This is used to generate the differences in SignIn/SignUp dynamically based on the application. 

group here uses 'system' which seems scary to let anyone sign up via a system access, but its the only group in set in foam.

Reply:
1) hmmm ... model the loginVariables... I like it. Good call.
2) to your point on the second paragraph. group and imgPath don't change- hardcoded. But since the new plan is to model this, then moot point.
3) props on loginView, doesn't work, these values change based on application, thus need applicationControllers to define them. 

But with Model, I see this working out to be a model, a dao, and journals ... we could even permission the selects - potentially opening up switching views from one applicationController and/or using crunch. I'm even now thinking, move some of the disableFieldBooleans_ to this jrl and just do a select on this loginVarDao when handling tokens. ... hmmm

Reply:
^ Can do later: https://nanopay.atlassian.net/browse/CPF-3588